lib/hg/messenger/bot.rb

```diff
@@ -222,9 +222,12 @@ def queue_postback(postback)
             raw_postback['postback'] = payload
           # Else, it may be a referral, but Get Started postback has been received
           elsif raw_postback['postback']['referral']
-            payload = raw_postback['postback']['referral']['ref']
-            # Parse the payload and set to 'postback'
-            raw_postback['postback'] = JSON.parse(payload)
+            # Parse the referral payload
+            payload = JSON.parse(raw_postback['postback']['referral']['ref'])
+            # Include onboarding param
+            payload['payload']['params'].merge!({ 'onboarding' => 'true' })
```

Comment:
Should this be `'true'`, or would `true` suffice? You can also drop the braces, since this is a hash argument.

I actually caught both of those with the Code Climate Firefox add-on, which is pretty handy: https://codeclimate.com/browser-extension/

![image](https://user-images.githubusercontent.com/2220/35937879-2e200c34-0c0d-11e8-8d89-9643f06d7927.png)

Reply:
I actually made these fixes and am testing now...